Firestore/Example/Tests/SpecTests/FSTSpecTests.mm

```diff
@@ -675,9 +679,19 @@ - (void)validateActiveTargets {
       XCTAssertEqualObjects(Describe(actual.resumeToken), Describe(queryData.resumeToken));
     }
 
-    [actualTargets removeObjectForKey:targetID];
-  }];
-  XCTAssertTrue(actualTargets.count == 0, "Unexpected active targets: %@", actualTargets);
+    actualTargets.erase(targetID);
```

Comment:
Since you've already got the iterator (kv) you could just erase that instead. Very optional.

Reply:
Yep, my mistake. I mistook kv for an iterator for some reason. (Despite the "kv" name.)